src/data/store.js

```diff
@@ -2,55 +2,16 @@ import { applyMiddleware, createStore } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
 import { createLogger } from 'redux-logger';
-import { createMiddleware } from 'redux-beacon';
-import Segment, { trackEvent, trackPageView } from '@redux-beacon/segment';
-import { routerMiddleware, LOCATION_CHANGE } from 'react-router-redux';
-import { FETCH_CSV_REQUEST } from './constants/csv';
-import { PAGINATION_REQUEST, SORT_REQUEST } from './constants/table';
 
-import apiClient from './apiClient';
-import history from './history';
 import reducers from './reducers';
 
 const loggerMiddleware = createLogger();
-const routerHistoryMiddleware = routerMiddleware(history);
 
-const eventsMap = {
```

Comment:
Just leaving a note for additional context on the removal of this `eventsMap` and `segmentMiddleware` below. We are intentionally removing tracking of the CSV downloads and table pagination/sorting as they are not actually used downstream for any product analytics afaik.

More importantly, we'd rather do analytics the right way with frontend-platform rather than dispatching these analytics events using `@redux-beacon/segment` as we were doing here.